.github/repo_policies/BOT_APPROVED_FILES

```diff
@@ -8,3 +8,4 @@ ic-os/hostos/context/docker-base.prod
 ic-os/setupos/context/docker-base.dev
 ic-os/setupos/context/docker-base.prod
 testnet/mainnet_revisions.json
+mainnet-canisters.json
```

Comment:
@aterga should this file also be in the `testnet/` directory?

Reply:
I would do it the other way around: move `testnet/mainnet_revisions.json` to `mainnet_revisions.json` (and ideally give it a better name). Then later we can drop the whole `testnet/` directory.

Reply:
I don't have a string opinion.

* Historically we had testnet/mainnet_revisions.json
* Then mainnet-canisters.json was added
* At this point the "testnet" directly is basically a collection of old stuff (doesn't mean unimportant), so Bas's suggestion makes sense to me to refactor that.